composer.json

```
@@ -46,11 +47,17 @@
     "sort-packages": true,
     "allow-plugins": {
       "dealerdirect/phpcodesniffer-composer-installer": true,
-      "roots/wordpress-core-installer": true
+      "roots/wordpress-core-installer": true,
+      "cweagans/composer-patches": true
     }
   },
   "extra": {
-    "wordpress-install-dir": "vendor/roots/wordpress"
+    "wordpress-install-dir": "vendor/roots/wordpress",
+    "patches": {
+      "google/apiclient-services": {
+        "Fixes for empty filter": "patches/google-api-client-services.diff"
```

Hmm, we already have a file in `patches` called `@wordpress+data+4.27.3.patch`, indicating it's a patch for `@wordpress/data` version `4.27.3`.

Can we rename the file we're introducing for consistency, so this it includes the version of `google-api-client-services` that it's patching (`v0.355.0`), and has the suffix `patch` instead of `diff`?

```suggestion
        "Fixes for empty filter": "patches/google-api-client-services-v0.355.0.patch"
```